docs/stratis.txt

```diff
@@ -72,6 +72,8 @@ filesystem rename <pool_name> <fs_name> <new_name>::
 blockdev list [pool_name]::
 	 List all blockdevs that make up the specified pool, or all pools, if
 	 no pool name is given.
+key list::
+     List all Stratis key-descriptions in the kernel keyring that can be used for encryption.
```

Comment:
I think you should actually leave out "Stratis", because we don't use that in any of the other man page entries for key operations.

Reply:
Resolved.